app/services/IdentityService.scala

```diff
@@ -14,6 +13,8 @@ import play.api.libs.ws.{WS, WSRequest, WSResponse}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
+case class UserId(id:String)
```

Comment:
Minor performance optimisation: why not use a [value class](http://docs.scala-lang.org/overviews/core/value-classes.html) here?

```scala
case class UserId(id:String) extends AnyVal
```